packages/webpack-config/webpack/webpack.common.js

```diff
@@ -106,24 +107,7 @@ module.exports = function(env) {
     exclude: locations.template.folder,
   };
 
-  // This method intercepts modules being referenced in react-native
-  // and redirects them to web friendly versions in expo.
-  function getWebModule(initialRoot, moduleName) {
-    return function(res) {
-      if (res.context.includes('node_modules/react-native/')) {
-        res.request = locations.includeModule(initialRoot + moduleName);
-      }
-    };
-  }
-
-  function useWebModule(modulePathToHiJack, redirectPath, initialRoot = 'expo/build/web/') {
-    return new webpack.NormalModuleReplacementPlugin(
-      new RegExp(modulePathToHiJack),
-      getWebModule(initialRoot, redirectPath)
-    );
-  }
-
-  const publicPath = '/';
+  const publicPath = ''.replace(/\/$/, '');
```

Comment:
This is just an empty string `''`, is that on purpose?

Reply:
We may be adding `publicUrl` for export right? If so then this will prevent `//` in the index.html. This looks a little strange out of context because I split it into 2 PRs.